site/themes/embark/layout/archive.swig

```diff
@@ -44,7 +44,7 @@
     {% endfor %}
   </section>
   <div class="o-container o-distance o-center">
-    <a href="/news" class="c-button" title="{{__('archives.back_to_blog_link.title')}}">{{__('archives.back_to_blog_link.label')}}</a>
+    <a href="https://blog.embarklabs.io" class="c-button" title="{{__('archives.back_to_blog_link.title')}}">{{__('archives.back_to_blog_link.label')}}</a>
```

Comment:
We need to remove the domain here, otherwise this link won't work anymore when spinning up the site without internet connection.

Reply:
it's not suppose since the blog is in a different repo now